lang/rustup/Makefile

```diff
@@ -0,0 +1,64 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rustup
+PKG_VERSION:=1.22.1
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rust-lang/rustup.git
+PKG_SOURCE_DATE:=2020-07-18
+PKG_SOURCE_VERSION:=ac1a8c415b5c2886275ca92a3a8f3a4cd0587711
+PKG_HASH:=46c1cf800e48f43bf0b3e19bf1eb2ed414106e5ff2647cf7858286489454acf2
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+include ./rustc-triple.mk
+
+define Package/rustup
+    SECTION:=lang
+    CATEGORY:=Languages
+    TITLE:=Rust Programming Language Installer
+    URL:=https://www.rust-lang.org/
+endef
+
+define Package/rustup/description
+ The rustup package installs the Rust Programming Language (rustc/cargo)
+ for use with packages that require Rust to compile.
+
+ This is a HOST ONLY package; no installed files to the target device.
```

Comment:
What about BUILDONLY? ;-) Two spaces as indentation here as well.

Reply:
There is also `BUILDONLY:=1`; it seems host-only tools include both of these.

Reply:
PKG_HOST_ONLY it is.. BUILDONLY I don't believe will call Compile/Install